src/routes/explore/+page.svelte

```diff
@@ -1 +1,242 @@
-<h1>Explore</h1>
\ No newline at end of file
+<h1>
+Explore the ecosystem
+</h1>
+
+<h2>
+    Audio
+</h2>
+    <h3>
+        <a href="https://www.fountain.fm/">Fountain</a> - The podcast app that pays.
```

Comment:
I think I have a suggestion on how to do this better. I'll try and create a simple POC to show you.